pkg/lib/cockpit-components-shutdown.jsx

```diff
@@ -214,7 +214,7 @@ export class ShutdownModal extends React.Component {
                                                 locale={timeformat.dateFormatLang()}
                                                 weekStart={timeformat.firstDayOfWeek()}
                                                 onBlur={this.calculate}
-                                                onChange={(d, ds) => this.updateDate(d, ds)}
+                                                onChange={(_, d, ds) => this.updateDate(d, ds)}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18222-20230123-142319-6860d3f9-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)